app/demo-hifive-inventor/app.toml

```diff
@@ -6,25 +6,41 @@ stacksize = 896
 
 [kernel]
 name = "demo-hifive-inventor"
-requires = { flash = 16752, ram = 3232 }
+requires = { flash = 16784, ram = 3472 }
 features = []
 
 [tasks.jefe]
 name = "task-jefe"
 priority = 0
 max-sizes = { flash = 16384, ram = 2048 }
 start = true
-features = ["semihosting-riscv"]
+features = ["log-stringbuf"]
```

Comment:
This is not related to plic. It should be a separate change.

Reply:
Agreed, will fix.